action_tutorials/action_tutorials_cpp/README.md

```diff
@@ -0,0 +1,59 @@
+# Action Server
+
+In the constructor for `FibonacciActionServer`, an action server is created with callbacks that are called when a goal is received, when the goal is cancelled and when the goal is accepted:
+
+```
```

Comment:
It would be nice to add syntax highlighting for the code blocks by adding `cpp` after the three ticks, here and elsewhere. For more info see here: https://support.codebasehq.com/articles/tips-tricks/syntax-highlighting-in-markdown.